tox.ini

```diff
@@ -13,12 +13,10 @@ deps =
     py3: flake8
     py3: flake8-docstrings
     py3: flake8-import-order
-    mypy
```

Comment:
Sorry for the misunderstanding, but we do want to run mypy from tox. My comment was that in the python client repo, there were pytests that also ran mypy during the tests, which I thought is why it shows up in tests_require given that tox has its own deps list.

Reply:
Ah I see, my bad. I thought we might want it but figured you had a reason for removing it. That makes much more sense. I'll update 😄